packages/react-percy-test-framework/src/interfaces/getCommonInterface.js

```diff
@@ -25,15 +27,20 @@ export default function getCommonInterface(suites) {
       suites.unshift(suite);
       if (typeof fn === 'function') {
         await fn.call(suite);
-      } else {
-        throw new Error(`Suite "${suite.fullTitle()}" was defined but no callback was supplied.`);
       }
       suites.shift();
       return suite;
     },
     snapshot(title, options, fn) {
       const snapshot = new Snapshot(title, options, fn);
       suites[0].addSnapshot(snapshot);
+      const snapshotFullTitle = snapshot.fullTitle();
+      if (snapshotNames[snapshotFullTitle]) {
+        throw new Error(
+          `A snapshot named \`${snapshotFullTitle}\` has already been added, please use a different name`,
```

Comment:
Yeah that's fine, that's a nice-to-have. It might be nice to recommend specific ways of handling it, like using a wrapping `suite` or something (just thinking out loud).